src/store/api/gateway/safeOverviews.ts

```diff
@@ -125,22 +125,25 @@ export const safeOverviewEndpoints = (
     'gatewayApi'
   >,
 ) => ({
-  getSafeOverview: builder.query<
-    SafeOverview | undefined,
-    { safeAddress: string; walletAddress?: string; chainId: string }
-  >({
-    async queryFn({ safeAddress, walletAddress, chainId }, { getState }) {
-      const state = getState()
-      const currency = selectCurrency(state as RootState)
-
-      try {
-        const safeOverview = await batchedFetcher.getOverview({ chainId, currency, walletAddress, safeAddress })
-        return { data: safeOverview }
-      } catch (error) {
-        return { error: { status: 'CUSTOM_ERROR', error: (error as Error).message } }
-      }
+  getSafeOverview: builder.query<SafeOverview | null, { safeAddress: string; walletAddress?: string; chainId: string }>(
+    {
+      async queryFn({ safeAddress, walletAddress, chainId }, { getState }) {
+        const state = getState()
+        const currency = selectCurrency(state as RootState)
+
+        if (!safeAddress) {
+          return { data: null }
```

Comment:
Had to change the return type to `null` because `undefined` is not allowed as return type for `queryFn`